packages/dialtone-vue2/components/lazy_show/lazy_show.vue

```diff
@@ -11,7 +11,7 @@
       v-on="$listeners"
     >
       <!-- @slot Slot for main content -->
-      <slot v-if="initialized" />
```

Comment:
oh, that makes sense. Will probably have to find another ways of testing this, unless suspense works better with the automation. Thanks!

Reply:
I've been checking this out and even without the "initialized" logic, the component is not rendered the first time, because the show prop is still false. So I don't think this is actually making any performance difference.
Without the `v-if="initialized"` I can see that the DOM is only updated when the "show" property is true:
![init-vue3](https://github.com/dialpad/dialtone/assets/24460973/369df436-dfd0-4b04-a9c2-0966ff04f956)

The slot from lazy-show is inside the div we see appearing and disappearing from the DOM in the gif:
<img width="968" alt="image" src="https://github.com/dialpad/dialtone/assets/24460973/0ec98d4c-0eef-4b1f-8b7d-cf54d6b778ec">

I also tried this is vue 2 and it works the same way.

For what I can see, Suspense is for managing lazy loading of async dependencies, which is not what lazy show seems to be doing.

Reply:
Disregard my previous comment, the slot content is actually being rendered (with `display: none`), until the first time the anchor is hovered. It is inside the popover:
<img width="967" alt="image" src="https://github.com/dialpad/dialtone/assets/24460973/dd148ecd-bacb-4472-95da-7c98ac1093d8">

When doing the first hover, tippy moves this to the body element. So lazy-show is doing a render optimization.

The problem with the test was that I was expecting the elements to be inside the wrapper, while they are on the body.
This explains also what @iropolo pointed out about the tests, they are passing even without the timers because the slot is being rendered only with `display: none`.

I will revert the changes to `lazy_show` and test that the element is present in the body element.

Reply:
Added ticket to look into the lazy-show further. https://dialpad.atlassian.net/browse/DLT-1477

once we get tests passing here, we're good to move forward.

Reply:
Fixed in latest changes